cmd/tink-worker/cmd/container.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"context"
+	"path"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/client"
+	"github.com/pkg/errors"
+)
+
+type containerAction struct {
+	actionPid      string
+	actionVolumes  string
+	actionEnv      []string
+	actionName     string
+	volumeBindings []string
+}
+
+var containerState = map[int32]string{
+	0: "SUCCESS",
+	1: "FAILED",
+	2: "TIMEOUT",
+}
+
+// createContainer creates container and returns containerid on success else returns error.
+func createContainer(ctx context.Context, cli client.ContainerAPIClient, cActions containerAction, cmd []string, registry string, imageName string, captureLogs bool) (string, error) {
```

Comment:
another thought on naming. If this code is moved to a new pkg, possibly named `container`, then a consumer of this function would call `container.CreateContainer`. Would we need the `Container` part of the name? maybe we just use `container.Create`?

Reply:
Also, this function mixes some bare parameters, like `cmd []string, registry string, imageName string`, with a custom struct `cActions containerAction`. I'm curious about the thought behind this?